.github/workflows/build_and_test.yml

```diff
@@ -617,7 +617,11 @@ jobs:
         python3.11 -m pip install 'black==23.9.1' 'protobuf==5.28.3' 'mypy==1.8.0' 'mypy-protobuf==3.3.0'
         python3.11 -m pip list
     - name: Python CodeGen check
-      run: ./dev/connect-check-protos.py
+      run: |
```

Comment:
It has not been merged with `connect-check-protos.py` in order to avoid increasing the complexity of the script, If merging them together would be better, please let me know.